client-app/src/examples/contact/svc/ContactService.ts

```diff
@@ -52,6 +53,11 @@ export class ContactService extends HoistService {
         const {userFaves} = this,
             isFavorite = userFaves.includes(id);
 
+        XH.setPref(PERSIST_APP.prefKey, {
```

Comment:
As noted above, I will need some help with persistence management.

Existing (desktop) contacts is saving grid / tile view, but is NOT saving favorites. I see the userFaves being wired up, but then the persistence code never gets called subsequently when the state changes on an item. I am unsure why.

So this resolves it - call persistence manually to save the value. But ultimately I should understand why its not working by default and resolve it that way.